daemon/config.go

```diff
@@ -124,7 +124,7 @@ func (c *Config) SetKVBackend() error {
 	case kvstore.Etcd:
 		log.Infof("using etcd as key-value store")
 		c.EtcdConfig = &etcdAPI.Config{}
-		c.EtcdConfig.Endpoints = []string{"http://127.0.0.1:4002"}
+		c.EtcdConfig.Endpoints = []string{"http://127.0.0.1:4001"}
```

Comment:
https://coreos.com/etcd/docs/latest/v2/configuration.html

    The official etcd ports are 2379 for client requests, and 2380 for peer communication. Some legacy code and 
    documentation still references ports 4001 and 7001

I think we should even change it to 2379 then.